system/Database/Forge.php

```diff
@@ -330,6 +337,61 @@ public function addField($field)
 	}
 
 	//--------------------------------------------------------------------
+        
+        /**
+	 * Add Foreign Key
+	 *
+	 * @param    array $field
+	 *
+	 * @return    CI_DB_forge
```

Comment:
In other methods (addField, addKeys) is the same.

Whats is the correct return?  \CodeIgniter\Database\Forge

This return \CodeIgniter\Database\Postgre\Forge or \CodeIgniter\Database\MySQLi\Forge